core/types/transaction.go

```diff
@@ -33,6 +33,13 @@ import (
 	common2 "github.com/harmony-one/harmony/internal/common"
 )
 
+// RPCTransactionError ..
+type RPCTransactionError struct {
+	TxHashID             string `json:"tx-hash-id"`
```

Comment:
are we consistent with all the json string?  I saw some complaint about inconsistent json field name in the API.

Reply:
JSON APIs should be kebab-case, but not all places do that, can clean up the snake_case JSON APIs later